apps/omg/lib/omg/application.ex

```diff
@@ -20,9 +20,11 @@ defmodule OMG.Application do
 
   use Application
   alias OMG.Alert.AlarmHandler
+  require Logger
 
   def start(_type, _args) do
     :ok = AlarmHandler.install()
+    {_, _} = Logger.add_backend(Sentry.LoggerBackend)
```

Comment:
Not sure how to do this the right way but the [documentation](https://github.com/getsentry/sentry-elixir#capture-crashed-process-exceptions) and [an issue](https://github.com/getsentry/sentry-elixir/issues/345) shows that it's only needed to be loaded once.